os_stub/spdm_device_secret_lib_sample/key_pair.c

```diff
@@ -30,147 +30,191 @@ typedef struct {
     uint8_t public_key_info[SPDM_MAX_PUBLIC_KEY_INFO_LEN];
 } libspdm_key_pair_info_t;
 
-#ifndef LIBSPDM_MAX_KEY_PAIR_COUNT
-#define LIBSPDM_MAX_KEY_PAIR_COUNT 16
-#endif
+#define LIBSPDM_MAX_KEY_PAIR_COUNT 9
 
 libspdm_key_pair_info_t m_key_pair_info[LIBSPDM_MAX_KEY_PAIR_COUNT];
 
-bool g_need_init_key_pair_info = true;
+uint8_t m_total_key_pair_count = 0;
```

Comment:
Can make this and line 35 `static`. I can document the variable naming conventions somewhere. In particular
- `m_*` -> file scope, no external linkage
- `g_*` -> file scope, with external linkage